File: 2 курс/Веб-программирование/lab2/lab2/src/main/java/utility/Point.java
package utility;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;



@ToString
@Getter
@NoArgsConstructor
public class Point {
    private String x;
    private String y;
    private String r;
    private String status;
    private String currentTime;
    private String executionTime;
    @Setter
    private boolean valid;
    @Setter
    private String validationMessage;

    public Point(String x, String y, String r, String status, String currentTime, String executionTime) {
        this.x = x;
        this.y = y;
        this.r = r;
        this.status = status;
        this.currentTime = currentTime;
        this.executionTime = executionTime;
    }
}

File: 2 курс/Веб-программирование/lab4/front/src/app/models/Point.ts
export class Point {

  private readonly _x: string
  private readonly _y: string
  private readonly _r: string
  private readonly _status: string
  private readonly _currentTime: string
  private readonly _executionTime: string

  constructor(x: string, y: string, r: string, status: string, currentTime: string, executionTime: string) {
    this._x = x
    this._y = y
    this._r = r
    this._status = status
    this._currentTime = currentTime
    this._executionTime = executionTime
  }


  get x(): string {
    return Number(this._x).toFixed(2)
  }

  get y(): string {
    return Number(this._y).toFixed(2)
  }

  get r(): string {
    return this._r
  }

  get status(): string {
    return this._status
  }

  get currentTime(): string {
    return this._currentTime
  }

  get executionTime(): string {
    return this._executionTime
  }
}
